pages/SearchPage.vue

```diff
@@ -1,5 +1,257 @@
 <template>
-  <div>
-    Search
+  <div v-if="authModule.isAuthenticated">
+    <h1 class="text-4xl font-bold text-gray-700 mt-10 ml-10">
+      Search
+    </h1>
+    <div
+      id=""
+      class="ml-10 mt-6 flex items-center text-lg"
+    >
+      <a
+        href="#"
+        class="text-blue-800 mr-5 font-semibold align-middle hover:text-blue-900 transition duration-150"
+        @click="clearFilters()"
+      >Clear Filters</a>
+      <div class="relative columnsDropdown">
+        <button
+          class="transition ease-in-out delay-120 hover:-translate-y-2 hover:scale-115 duration-300
+          px-4 py-2 border border-gray-300 bg-white text-black w-60 ml-5 mt-1"
+          @click="toggleDropdown('columnsDropdown')"
+        >
+          <span class="mr-2">Columns to Show</span>
+          <font-awesome-icon
+            :icon="['fas', columnsDropdown ? 'arrow-up' : 'arrow-down']"
+            :class="columnsDropdown ? 'rotate-180' : 'rotate-0'"
```

Comment:
Why do you both change the icon (arrow-up / arrow-down) and rotate it -- don't you just need to do one of those?

Reply:
Yes, removed the arrow-up and made the rotate function 360 deg instead

Reply:
OK, when I see the new code then maybe it will make sense to me.

But why rotate 360 degrees? Isn't that the same as no rotation?

Reply:
Essentially, yes. When the user clicks on a button, this arrow is rotated 360 deg clockwise with a transition and the dropdown opens if closed. It's rotated 360 counter-clockwise when clicked again to close it.

Reply:
So it's simply a neat animation?

In other BCROS apps, we use down arrow to indicate a menu open action, and an up arrow to indicate a menu close action, ie:

![image](https://github.com/bcgov/name-examination/assets/10002889/8bef339b-51c8-4fa2-acaa-7f3a8fc1ee7d)

It would be nice to be consistent.

Reply:
Ok, reverting it to rotate-180 makes it consistent. Implemented

Reply:
Haha, sorry but I appreciate it!

Reply:
OK, I looked at the latest code and now it makes sense to me :)
Resolving.